examples/example_notebook.ipynb

```diff
@@ -13,7 +13,7 @@
    "source": [
     "Make sure to have installed the quart_solar_forecast package: \n",
     "```\n",
-    "pip install quartz-solar-forecast\n",
+    "pip install git+https://github.com/openclimatefix/Open-Source-Quartz-Solar-Forecast.git@tryolabs-hourly-predictions\n",
```

Comment:
Im suprised you have to use this branch? What was the error on the main branch? Perhaps the model has now been upgrade now?

Reply:
The model 0.4.0 is there, https://github.com/openclimatefix/Open-Source-Quartz-Solar-Forecast/blob/main/quartz_solar_forecast/models/model-0.4.0.pkl, but perhaps its somehow not working.

Reply:
When I pip install without specifying the branch, I ran into the error that model-0.4.0 is not available when I ran forecast. I checked the package folder and there's only model-0.3.0.

Reply:
<img width="993" alt="Screenshot 2024-07-03 at 4 06 01 PM" src="https://github.com/openclimatefix/Open-Source-Quartz-Solar-Forecast/assets/14258987/c356410b-94a9-47b3-9463-34f3d2a307e5">
Here's the error

Reply:
Ah, i see the error, ive put it here - https://github.com/openclimatefix/Open-Source-Quartz-Solar-Forecast/issues/140

Reply:
can your merge in to latest main branch, and see if it works with "pip install quartz-solar-forecast" now?

Reply:
I just installed the latest package and the errors went away. Also changed all the 'wh' back to 'kw' because the new package uses power_kw